features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -161,6 +161,7 @@ internal open class RumViewScope(
         event: RumRawEvent,
         writer: DataWriter<Any>
     ): RumScope? {
+        updateGlobalAttributes(sdkCore, event)
```

Comment:
With this new logic we are not taking into account the current value of global attributes when we send the view update on `stop` view. Before we were always taking the last value. This might change the way the sdk works in our clients apps so we need to debate more on this one. @0xnm @plousada @xgouchet